concepts/05 Widgets/Scheduler/030 Appointments/020 Add Appointments/020 API.md

```diff
@@ -76,4 +76,94 @@ To add an appointment to the [data source](/api-reference/10%20UI%20Widgets/dxSc
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            :data-source="dataSource"
+            :current-date="currentDate"
+        />
+        <DxButton
+            text="Add"
+            @click="addAppontment"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxScheduler from 'devextreme-vue/scheduler';
+
+    export default {
+        components: {
+            DxScheduler,
+            DxButton
+        },
+        data() {
+            return {
+                dataSource: [{
+                    text: "Planning",
+                    startDate: new Date(2016, 4, 25, 9, 00),
+                    endDate: new Date(2016, 4, 25, 9, 30)
+                }, 
+                // ...
+                ],
+                currentDate: new Date(2016, 4, 25),
+            },
+            methods: {
+                addAppontment: (e) {
+                    e.component.addAppointment({
+                        text: "Website Re-Design Plan",
+                        startDate: new Date(2016, 4, 25, 9, 30),
+                        endDate: new Date(2016, 4, 25, 11, 30)
+                    })
+                }
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import Scheduler from 'devextreme-react/scheduler';
+
+    const addAppointment = (e) => {
+        e.component.addAppointment({
+            text: "Website Re-Design Plan",
+            startDate: new Date(2016, 4, 25, 9, 30),
+            endDate: new Date(2016, 4, 25, 11, 30)
+        })
+    }
+    const dataSource = [{
+        text: "Planning",
+        startDate: new Date(2016, 4, 25, 9, 00),
+        endDate: new Date(2016, 4, 25, 9, 30)
+    }, 
+    // ...
+    ];
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Scheduler
```

Comment:
`render()` cannot return more than one element. You need to wrap them in `<React.Fragment>`.